src/components/pages/Publish/FormActions.tsx

```diff
@@ -1,40 +1,46 @@
 import React, { FormEvent, ReactElement } from 'react'
 import { useOcean } from '../../../providers/Ocean'
 import { useWeb3 } from '../../../providers/Web3'
+import Alert from '../../atoms/Alert'
 import Button from '../../atoms/Button'
 import styles from './FormActions.module.css'
 
 export default function FormActions({
   isValid,
-  resetFormAndClearStorage
+  resetFormAndClearStorage,
+  walletDisclaimer
 }: {
   isValid: boolean
   resetFormAndClearStorage: (e: FormEvent<Element>) => void
+  walletDisclaimer: string
 }): ReactElement {
   const { ocean, account } = useOcean()
   const { isChainIdAllowed } = useWeb3()
 
   return (
-    <footer className={styles.actions}>
-      <Button
-        style="primary"
-        type="submit"
-        disabled={
-          !ocean ||
-          !account ||
-          !isValid ||
-          status === 'empty' ||
-          !isChainIdAllowed
-        }
-      >
-        Submit
-      </Button>
-
-      {status !== 'empty' && (
-        <Button style="text" size="small" onClick={resetFormAndClearStorage}>
-          Reset Form
+    <footer className={styles.actionsFooter}>
+      <div className={styles.actions}>
+        <Button
+          style="primary"
+          type="submit"
+          disabled={
+            !ocean ||
+            !account ||
+            !isValid ||
+            status === 'empty' ||
+            !isChainIdAllowed
+          }
+        >
+          Submit
         </Button>
-      )}
+
+        {status !== 'empty' && (
+          <Button style="text" size="small" onClick={resetFormAndClearStorage}>
+            Reset Form
+          </Button>
+        )}
+      </div>
+      <Alert state="info" text={walletDisclaimer} />
```

Comment:
suggestion(non-blocking): I like that we use the `info` state here. However checking the preview it seems that warning & info use the same color scheme. Would expect something like our blue or gray for information, and use the orange for warning only.

I guess not high prio, only fix if this is small scope

Reply:
I agree, I'll check if it's used somewhere else and update it with the gray color scheme, should take just a few minutes.

Reply:
Update: I checked and the `info' state is used in a few other places. The idea would be to turn those places to `warning` (to keep the current color) and just keep info for the new disclaimer under the buy button, would it be ok for you?

Reply:
Yes, sounds right! Let's update both new disclaimers to info scheme (publishing and consuming)